### ISSUE TITLE
PB Tabs Tray icon isn't read properly when no Private tabs are open

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#16415]
## Prerequisites
Have two open tabs.
Talkback should be enabled
Grid view in Tabs Tray should be enabled
❗ Make sure you don't have any Private open tabs

## Steps to reproduce
1. Open The Tabs Tray
2. Tap the PB icon 

### Expected behavior
Should read "Private Tabs, Tab"

### Actual behavior
"Private Tabs, Tab, 2 of 2"

### Device information

* Android device: Google Pixel 3a (Android 11)
* Fenix version: Nightly Build 201106

### Notes
► Screenshot
![rsz_screenshot_20201106-114448](https://user-images.githubusercontent.com/51314259/98353811-852c8480-2028-11eb-93c8-431c3dca120a.png)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-8670)

**Comment by mcarare** [comment]
This is the expected behavior, as it relates to the number of tabs in the bottom sheet. There might be some confusion as the tabs word is used for both Fenix tabs and bottom sheet tabs. See also: https://github.com/mozilla-mobile/fenix/issues/16417#issuecomment-723018775.
Maybe UX should weigh in on this.

**Comment by AndiAJ** [comment]
From my point of view, if there are **no Private open tabs**, the PB icon shouldn't count anything, should only read Should read "Private Tabs, Tab".

**Comment by topotropic** [comment]
Did the behavior change? When I select Private Tabs, I just get  "Private Tabs" via Talkback.

**Comment by mcarare** [comment]
@topotropic It says "2 of 2" for me, that IMO that is expected as it refers to the tabs in the tab tray, not the browser tabs. Just wanted UX to weigh in if maybe should be a semantic difference between these  type of "tabs".

**Comment by topotropic** [comment]
> @topotropic It says "2 of 2" for me, that IMO that is expected as it refers to the tabs in the tab tray (normal and private), not the browser tabs. Just wanted UX to weigh in if maybe should be a semantic difference between these type of "tabs".

Ok, I understand, I agree that this is potentially confusing. What options do we have here? Is it recommended/expected that we follow this format "Private Tabs, Tab, 2 of 2"? Thanks!

**Comment by mcarare** [comment]
AFAIK we should not change the default a11y announcements as it could confuse a11y users. Maybe someone from a11y team can also weigh in on this. 
Maybe Fenix tabs could be referred to as "browser tabs"?

**Comment by topotropic** [comment]
Ok, I asked the a11y team

**Comment by topotropic** [comment]
The behavior is correct – we could update it so that it says when Private tabs are selected "Selected Private Tabs, 2 of 2" or if not selected then just "Private tabs, 2 of 2" instead of "Private Tabs, Tab, 2 of 2" as a small improvement.

**Comment by boek** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1812893

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
